### PR TITLE
docs: application configuration

### PR DESCRIPTION
**Pull request by technicallyty**
adding docs for application config

### CODE REVIEW

File: docs/developers/integration.mdx [hunk 1]
```diff
@@ -34,9 +37,75 @@ Integrating Connect into your Cosmos SDK application requires a few simple steps
...
+
+Finally, add a default configuration.
+
+```go config.go
+func DefaultConfig() (string, CustomAppConfig) {
+	serverConfig := serverconfig.DefaultConfig()
+    // edit serverConfig as needed
+
+	oracleCfg := oracleconfig.AppConfig{
+		Enabled:        false,
```
Comment (Eric-Warehime) [inline docs/developers/integration.mdx]: Don't we want to default the oracle to enabled?

File: docs/developers/integration.mdx [hunk 2]
```diff
@@ -34,9 +37,75 @@ Integrating Connect into your Cosmos SDK application requires a few simple steps
...
+		OracleAddress:  "localhost:8080",
+		ClientTimeout:  time.Second * 2,
+		MetricsEnabled: false,
```
Comment (Eric-Warehime) [inline docs/developers/integration.mdx]: Is there an issue with enabling this by default? Also, wasn't sure if we got rid of all of this separate metrics stuff? Shouldn't we just always attach oracle metrics to the app's telemetry?

File: docs/developers/integration.mdx (hunk 1 above)
Comment (technicallyty) [reply docs/developers/integration.mdx]: [switch to true](https://github.com/skip-mev/slinky/pull/660/commits/e042ab1c5d08bc6dbab13ee116dc513db28f4ad3)

File: docs/developers/integration.mdx (hunk 2 above)
Comment (technicallyty) [reply docs/developers/integration.mdx]: [switch to true](https://github.com/skip-mev/slinky/pull/660/commits/e042ab1c5d08bc6dbab13ee116dc513db28f4ad3)

Comment (technicallyty) [reply docs/developers/integration.mdx]: as for getting rid of separate metrics, im not sure. the app config in main currently looks like this:

```go
// AppConfig contains the application side oracle configurations that must
// be set in the app.toml file.
type AppConfig struct {
	// Enabled indicates whether the oracle is enabled.
	Enabled bool `mapstructure:"enabled" toml:"enabled"`

	// OracleAddress is the URL of the out of process oracle sidecar. This is
	// used to connect to the oracle sidecar.
	OracleAddress string `mapstructure:"oracle_address" toml:"oracle_address"`

	// ClientTimeout is the time that the client is willing to wait for responses
	// from the oracle before timing out.
	ClientTimeout time.Duration `mapstructure:"client_timeout" toml:"client_timeout"`

	// MetricsEnabled is a flag that determines whether oracle metrics are enabled.
	MetricsEnabled bool `mapstructure:"metrics_enabled" toml:"metrics_enabled"`

	// PriceTTL is the maximum age of the latest price response before it is considered
	// stale.
	PriceTTL time.Duration `mapstructure:"price_ttl" toml:"price_ttl"`

	// Interval is the time between each price update request.
	Interval time.Duration `mapstructure:"interval" toml:"interval"`
}
```

**Review by Eric-Warehime — APPROVED** [review]

File: docs/developers/integration.mdx [hunk 3]
```diff
@@ -34,9 +37,75 @@ Integrating Connect into your Cosmos SDK application requires a few simple steps
     </Step>
 </Steps>
 
+## Application Configuration
+
+The application node's configuration must be extended, so the oracle configuration can be read into the node through `app.toml`.
```
Comment (aljo242) [inline docs/developers/integration.mdx]:
```suggestion
The application node's configuration must be extended so the oracle configuration can be read into the node through `app.toml`.
```

**Review by aljo242 — APPROVED** [review]

**Pull request merged**